source/guide_snibox.rst

```diff
@@ -0,0 +1,216 @@
+.. highlight:: console
+
+.. author:: Maximilian F. Kapfer <maximilian.kapfer@gmail.com>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/snibox.png
+      :align: center
+
+######
+Snibox
+######
+
+Snibox is a self-hosted snippet manager. Developed to collect and organize code snippets. Supports various programming languages, markdown, plain text.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PostgreSQL_
+  * Ruby_
+  * supervisord_
+  * domains_
```

Comment:
These links currently point to nothing, as you can see in the [rendered guide](https://deploy-preview-952--uberlab.netlify.app/guide_snibox.html). Maybe like this?

```suggestion
  * :lab:`PostgreSQL <guide_postgresql>`
  * :manual:`Ruby <lang-ruby>`
  * :manual:`supervisord <daemons-supervisord>`
  * :manual:`domains <web-domains>`
```

Reply:
Exactly like this! Thanks